server/graphql/common/expenses.ts

```diff
@@ -1005,7 +1005,7 @@ export async function markExpenseAsUnpaid(req, ExpenseId, processorFeeRefunded):
     }
 
     const transaction = await models.Transaction.findOne({
-      where: { ExpenseId },
+      where: { ExpenseId, RefundTransactionId: null },
```

Comment:
Made this quick fix while I was there, steps were:
- Pay expense
- Mark it as unpaid
- Pay again
- Mark it as unpaid
- ==> See crash, "Transaction already refunded"